### PR TITLE
fix(GlobalHeader): BAM profile menu changes Revert

### PR DESCRIPTION
**Pull request by jenesh**
### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: [ABC-123]
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](http://www.google.fr/ 'Named link title')  | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Portal       | [Portal Link](http://www.google.fr/ 'Named link title')  | [Portal Env](http://www.google.fr/ 'Named link title')   |
| Another Repo | [Another Link](http://www.google.fr/ 'Named link title') | [Another Env](http://www.google.fr/ 'Named link title')  |

### CODE REVIEW

File: packages/gamut-labs/src/GlobalHeader/GlobalHeaderItems.tsx [hunk 1]
```diff
@@ -283,12 +283,8 @@ export const freeProfile = (
 ): AppHeaderProfileDropdownItem => {
   const topSection = [profileMyProfile];
 
-  if (
-    !user.hideBusinessAccount ||
-    user.isBusinessAdmin ||
-    !user.isBusinessSsoUser
-  ) {
-    topSection.push(profileBusinessAccount);
+  if (user?.isBusinessAdmin || !user?.isBusinessSsoUser) {
```
Comment (jenesh) [inline packages/gamut-labs/src/GlobalHeader/GlobalHeaderItems.tsx:286]: Reverting to what it was before, PR for comparison ➡️  https://github.com/Codecademy/gamut/pull/2532/files

**Review by vonbarown — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.198-alpha.b00d89.0<br/>@codecademy/gamut-labs@33.3.2-alpha.b00d89.0<br/>@codecademy/styleguide@61.5.4-alpha.b00d89.0<br/></pre>

File: packages/gamut-labs/src/GlobalHeader/GlobalHeaderItems.tsx [hunk 2]
```diff
@@ -322,11 +318,7 @@ export const proProfile = (user: User): AppHeaderProfileDropdownItem => {
 
   topSection.push(profileMyHome);
 
-  if (
-    !user.hideBusinessAccount ||
-    user.isBusinessAdmin ||
-    !user.isBusinessSsoUser
-  ) {
+  if (user?.isBusinessAdmin || user?.isBusinessSsoUser) {
```
[the remark this answers is not in the document]
Comment (jenesh) [reply packages/gamut-labs/src/GlobalHeader/GlobalHeaderItems.tsx]: pushing the change now

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.198-alpha.862f44.0<br/>@codecademy/gamut-labs@33.3.2-alpha.862f44.0<br/>@codecademy/styleguide@61.5.4-alpha.862f44.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6318c2bd8df4c51b6e685765--gamut-preview.netlify.app>https://6318c2bd8df4c51b6e685765--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6318c2bd8df4c51b6e685765>Deploy Logs</a></p>

**Review by antoniablair — APPROVED** [review]
Lgtm

**Pull request merged**